engine/gamesys/src/gamesys/test/sound/set_sound.script

```diff
@@ -0,0 +1,17 @@
+function init(self)
+    self.updated = false
+end
+
+function update(self, dt)
+    -- update sound only once
+    if not self.updated then
+        local f = io.open("src/gamesys/test/sound_ext/replacement.wav")
+        local c  = f:read("*all")
```

Comment:
I think it is just "*a"

Reply:
Both "*a" and "*all" seem to work according to some posts on SO.

Roberto Ierusalimschy uses "*all" in his official doc: https://www.lua.org/pil/21.1.html

Reply:
Ah! Who am I to object to the wisdom of Roberto :-)